### PR TITLE
Instruct user to tap untapped official tap when running its commands

### PR DESCRIPTION
**Pull request by colindean**
This came up in https://github.com/Homebrew/homebrew-bundle/issues/1108 wherein a user had unwittingly untapped some official taps that get automatically tapped on first use and couldn't figure out why they couldn't use the associated command.

⚠️ I'm not really sure how best to test this. I think I could add something [here](https://github.com/Homebrew/brew/blob/ee93d7e08dbdc52d932572007bee0f5fb1d75cbe/Library/Homebrew/test/cmd/bundle_spec.rb#L4) that runs an empty check, then untaps the homebrew/bundle tap, then checks for an error when it runs the empty check again.

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [X] Have you successfully run `brew style` with your changes locally?
- [X] Have you successfully run `brew typecheck` with your changes locally?
- [X] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Library/Homebrew/brew.rb [hunk 1]
```diff
@@ -104,6 +104,13 @@
     possible_tap = Tap.fetch(possible_tap.first) if possible_tap
 
     if !possible_tap || possible_tap.installed? || Tap.untapped_official_taps.include?(possible_tap.name)
+      blocked_tap = possible_tap && Tap.untapped_official_taps.include?(possible_tap.name)
```
Comment (MikeMcQuaid) [inline Library/Homebrew/brew.rb:107]: Can you move the `Tap.untapped_official_taps.include?(possible_tap.name)` into a variable so it's only checked once?

File: Library/Homebrew/brew.rb [hunk 2]
```diff
@@ -104,6 +104,13 @@
...
+      if blocked_tap
+        [
+          "`brew #{cmd}` is unavailable because #{possible_tap.name} was manually untapped and cannot be retapped.",
+          "Run `brew tap #{possible_tap.name}` to reenable `brew #{cmd}`.",
+        ].each { |ln| onoe ln }
```
Comment (MikeMcQuaid) [inline Library/Homebrew/brew.rb:112]: Do a single `onoe` with `<~EOS` to handle multiple lines.

File: Library/Homebrew/brew.rb [hunk 3]
```diff
@@ -104,6 +104,13 @@
     possible_tap = Tap.fetch(possible_tap.first) if possible_tap
 
     if !possible_tap || possible_tap.installed? || Tap.untapped_official_taps.include?(possible_tap.name)
+      blocked_tap = possible_tap && Tap.untapped_official_taps.include?(possible_tap.name)
+      if blocked_tap
+        [
+          "`brew #{cmd}` is unavailable because #{possible_tap.name} was manually untapped and cannot be retapped.",
```
Comment (MikeMcQuaid) [inline Library/Homebrew/brew.rb:110]:
```suggestion
          "`brew #{cmd}` is unavailable because #{possible_tap.name} was manually untapped.",
```

**Comment by colindean** [comment]
🎉

**Pull request merged**